scripts/release/release.sh

```diff
@@ -146,7 +146,7 @@ function apply_cherry_picks() {
 function find_last_release() {
   local branch="${1:-HEAD}"
   local baseline="${2:-$(get_release_baseline "${branch}")}"
```

Comment:
Aren't the inner quotes superfluous here?

Reply:
They are not if branch contains a space or some random characters.